Guardfile

```diff
@@ -0,0 +1,35 @@
+guard 'livereload' do
```

Comment:
Why are you setting up Guard as part of this PR? 🤔 

Not a blocker but please stay in scope when working on stories. I'd rather you create a separate chore to deal with this if you really needed it